docs/sql-ref-syntax-qry-select-like.md

```diff
@@ -21,12 +21,14 @@ license: |
 
 ### Description
 
-A LIKE predicate is used to search for a specific pattern.
+A LIKE predicate is used to search for a specific pattern. LIKE predicate also supports multiple patterns with quantifiers include `ANY`, `SOME` and `ALL`.
 
 ### Syntax
 
 ```sql
 [ NOT ] { LIKE search_pattern [ ESCAPE esc_char ] | [ RLIKE | REGEXP ] regex_pattern }
+
+[ NOT ] { LIKE quantifiers ( search_pattern [ , ... ]) }
```

Comment:
`quantifiers` -> `[ ANY | SOME | ALL ]`?

Reply:
We need to describle quantifiers in detail. So I think `quantifiers` is better.